controllers/pod.go

```diff
@@ -107,6 +107,12 @@ func (r *AerospikeClusterReconciler) getRollingRestartTypePod(aeroCluster *asdbv
 		r.Log.Info("Aerospike resources changed. Need rolling restart")
 	}
 
+	// Check if podSpec is updated
+	if isPodSpecUpdatedInAeroCluster(aeroCluster, pod) {
```

Comment:
This is redundant, but produces more meaningful log messages than checksum check. LMK if you want me to get rid of it.